signer/src/transaction_coordinator.rs

```diff
@@ -170,13 +191,13 @@ where
     #[tracing::instrument(skip_all, name = "tx-coordinator")]
     pub async fn run(mut self) -> Result<(), Error> {
         tracing::info!("starting transaction coordinator event loop");
-        let mut signal_stream = self.context.as_signal_stream(&self.network);
+        let mut signal_stream = self.context.as_signal_stream(run_loop_message_filter);
 
         loop {
             match signal_stream.next().await {
```

Comment:
nit: `while let Some(message) = signal_stream.next().await {` like in the other run loop? also in `signer/src/transaction_signer.rs`

Reply:
Fixed in https://github.com/stacks-network/sbtc/pull/907/commits/5ecf544445e816671902ebb68dcda3fab7ba2545.